src/plugins/abrt-dump-journal-xorg.c

```diff
@@ -114,6 +114,7 @@ static void watch_journald(abrt_journal_t *journal, const char *dump_location, i
         .decorated_cb = abrt_journal_watch_extract_xorg_crashes,
         .decorated_cb_data = &watch_conf,
         .strings = xorg_strings,
+        .blacklisted_strings = NULL,
```

Comment:
Why do you need to touch the Xorg parsing in commit fixing a bug in the Kernel oops parsing?